commands/base.py

```diff
@@ -121,7 +134,7 @@ def check_permissions(self):
 
     def check_channels(self):
         cr = self.channels_required
-        if cr and len(cr) > 0:
+        if cr is not None and len(cr) > 0:
             if self.message.channel not in cr and None not in cr:
```

Comment:
I originally had it so that if we defined a list like `[ None ]` it would pass over that. I suppose that's redundant if we just make the list reference None in the first place.